src/main/java/frc/team2412/robot/util/AutonomousChooser.java

```diff
@@ -0,0 +1,58 @@
+package frc.team2412.robot.util;
+
+import edu.wpi.first.wpilibj.smartdashboard.SendableChooser;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.team2412.robot.Subsystems;
+import frc.team2412.robot.commands.autonomous.FollowTrajectoryCommand;
+
+public class AutonomousChooser {
+    private AutonomousTrajectories trajectories;
+
+    private SendableChooser<AutonomousMode> autonomousModeChooser = new SendableChooser<>();
+
+    public AutonomousChooser(AutonomousTrajectories trajectories) {
+        this.trajectories = trajectories;
+
+        autonomousModeChooser.setDefaultOption("Square Path Auto", AutonomousMode.SQUARE_PATH_AUTO);
+        autonomousModeChooser.addOption("Star Path Auto", AutonomousMode.STAR_PATH_AUTO);
+
+        SmartDashboard.putData("Choose Auto Mode", autonomousModeChooser);
+    }
+
+    public SendableChooser<AutonomousMode> getAutonomousModeChooser() {
+        return autonomousModeChooser;
+    }
+
+    private SequentialCommandGroup getSquarePathAutoCommand(Subsystems subsystems) {
+        SequentialCommandGroup command = new SequentialCommandGroup();
+
+        command.addCommands(
+                new FollowTrajectoryCommand(subsystems.drivebaseSubsystem, trajectories.getSquarePathAuto()));
+
+        return command;
+    }
+
+    private SequentialCommandGroup getStarPathAutoCommand(Subsystems subsystems) {
+        SequentialCommandGroup command = new SequentialCommandGroup();
+
+        command.addCommands(new FollowTrajectoryCommand(subsystems.drivebaseSubsystem, trajectories.getStarPathAuto()));
+
+        return command;
+    }
+
+    public SequentialCommandGroup getCommand(Subsystems subsystems) {
```

Comment:
Logging the path chosen would be good for forensics, troubleshooting and debugging.